docs/getting-started/os-support.rst

```diff
@@ -25,16 +25,16 @@ ScyllaDB Open Source
 
     The recommended OS for ScyllaDB Open Source is Ubuntu 22.04.
 
-+----------------------------+-------------+---------------+---------+---------------+--------------+--------------+
-| Linux Distributions        |Ubuntu       | Debian        | CentOS /| Rocky /       | Oracle Linux | Amazon Linux |
-|                            |             |               | RHEL    | RHEL          |              |              |
-+----------------------------+------+------+-------+-------+---------+-------+-------+-------+------+--------------+
-| ScyllaDB Version / Version |20.04 |22.04 |  10   |  11   | 7       |   8   |   9   |  7.6  | 8.1  | 2            |
-+============================+======+======+=======+=======+=========+=======+=======+=======+======+==============+
-|   5.4                      | |v|  | |v|  | |v|   | |v|   | |v|     | |v|   | |v|   | |v|   | |v|  | |v|          |
-+----------------------------+------+------+-------+-------+---------+-------+-------+-------+------+--------------+
-|   5.2                      | |v|  | |v|  | |v|   | |v|   | |v|     | |v|   | |x|   | |x|   | |x|  | |x|          |
-+----------------------------+------+------+-------+-------+---------+-------+-------+-------+------+--------------+
++----------------------------+-------------+---------------+---------------+--------------+--------------+
+| Linux Distributions        |Ubuntu       | Debian        | Rocky /       | Oracle Linux | Amazon Linux |
+|                            |             |               | RHEL          |              |              |
++----------------------------+------+------+-------+-------+-------+-------+-------+------+--------------+
+| ScyllaDB Version / Version |20.04 |22.04 |  10   |  11   |   8   |   9   |  7.6  | 8.1  | 2            |
++============================+======+======+=======+=======+=======+=======+=======+======+==============+
+|   5.4                      | |v|  | |v|  | |v|   | |v|   | |v|   | |v|   | |v|   | |v|  | |v|          |
++----------------------------+------+------+-------+-------+-------+-------+-------+------+--------------+
+|   5.2                      | |v|  | |v|  | |v|   | |v|   | |v|   | |x|   | |x|   | |x|  | |x|          |
```

Comment:
@annastuchlik Centos7 support was removed only from 5.4, in 5.2 we still support it

Reply:
And also oel76 was removed from 5.4 only